app/.env

```diff
@@ -0,0 +1,4 @@
+REACT_APP_API_VERSION=v1
+REACT_APP_API_LOCAL_SERVER_PORT=8080
+REACT_APP_API_LOCAL_HOST=localhost
```

Comment:
You can remove `REACT_APP_API_LOCAL_SERVER_PORT` and `REACT_APP_API_LOCAL_HOST` and rename `REACT_APP_API_LOCAL_SERVER_URL` to `REACT_APP_API_SERVER_URL`. the variable is for both local and production

Reply:
fix